Content.Server/Ghost/GhostSystem.cs

```diff
@@ -245,15 +245,19 @@ private IEnumerable<string> GetLocationNames()
             return null;
         }
 
-        private Dictionary<EntityUid, string> GetPlayerWarps(EntityUid except)
+        private Dictionary<EntityUid, string[]> GetPlayerWarps(EntityUid except)
         {
-            var players = new Dictionary<EntityUid, string>();
+            var players = new Dictionary<EntityUid, string[]>();
 
             foreach (var player in _playerManager.Sessions)
             {
                 if (player.AttachedEntity is {Valid: true} attached)
                 {
-                    players.Add(attached, EntityManager.GetComponent<MetaDataComponent>(attached).EntityName);
+                    TryComp<MindComponent>(attached, out var mind);
+
+                    string[] playerInfo = { EntityManager.GetComponent<MetaDataComponent>(attached).EntityName, mind?.Mind?.CurrentJob?.Name ?? "Unknow" };
+                    if (TryComp<MobStateComponent>(attached, out var state) && !state.IsDead())
+                        players.Add(attached, playerInfo);
```

Comment:
Wat.

Also why is this an array, it should just send them the string, also you spelt unknown wrong.

Reply:
Uh.... Aaaah... Becaaaauuuseee...
God told me it needed to be like this to atone for my sins.

In all honesty, I was a bit over my head while doing this and I thought I had to send only two arguments to players, which now that I look at it...

Reply:
Ok, now that I had some free time, I took a look over this again.

I did a string array because I didn't find a better way of sending down an extra string (name and role), without having to change all the types down the pipeline untill it reaches GhostTargetWindow.
I'm veeeeery new to C# and I couldn't think of any better options. Perhaps I should change the players Dictionary to something else that supports the Uid + 2 other strings?
Feel free to call me dummy, all tips are appreciated.

Reply:
I think you'll find my next commit a BIT LESS stupid.